src/test/calculate.test.js

```diff
@@ -0,0 +1,9 @@
+import calculate from '../logic/calculate';
```

Comment:
- [ ] I think there're still some scenarios that you haven't yet tested. Based on the requirement of this project, you are expected to test all functionalities and scenarios of this module.
- Kindly also considered the case where the buttonName is equal to i.e `buttonName === '+/-', '=', '.'(dot) `